### ISSUE TITLE
Compile error on Visual Studio 2017

### ISSUE THREAD

**Issue by carlcc** [rokups/rbfx#191]
Hi, have you tested building on windows? I failed, I cannot build neither building with the cmake generated .sln project nor use visual studio cmake support.

**Comment by eugeneko** [comment]
> Hi, have you tested building on windows?

We do have CI for Windows.
https://github.com/rokups/rbfx/runs/437832289

I recommend you to provide more details if you want to get things fixed.

**Comment by bvanevery** [comment]
I happened to build on VS 2017 yesterday with no issues.  Default settings only.  Used CMake GUI, did not enter any new information.  Did an out-of-source build.  CMake version 3.17.1.  Windows 10 latest greatest something.

Suggest you close this issue with "cannot reproduce".

**Comment by carlcc** [comment]
Sorry to leave this issue for such long time.

My previous visual studio 2017 installed a Chinese language pack, and indeed it failed to compile the source. The error information is also Chinese, so I did not past it here.

I installed vs2019 (and uninstalled 2017) the other day, without Chinese language pack, and I just compiled rbfx successfully.

I suspect the compile error was caused by the language pack?
